### PR TITLE
fix: display logo on Safari

### PR DESCRIPTION
**Pull request by DavideMininni-Fincons**
## Preflight Checklist

- [x] I have read the [Contributing Guidelines](https://github.com/lyne-design-system/lyne-components/blob/master/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/lyne-design-system/lyne-components/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [pull request tracker](https://github.com/lyne-design-system/lyne-components/pulls) for a Pull Request (PR) that matches the one I want to submit, without success.

## Issue

This PR Closes #

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)

See [Review Guidelines](../REVIEW.md) for more information what is checked during review process.

## Changes

Changes in this pull request:

- logo is not displayed on Safari (on all devices, Mac, iPhone and iPad); setting height and width on the SVG seems to solve the problem.

## Browsers

I tested the build on the following browsers:

- [ ] Firefox Desktop
- [ ] Chrome Desktop
- [ ] Edge Desktop
- [x] Safari Desktop
- [ ] Chrome Mobile
- [x] Safari Mobile

## Screen readers

I tested the build on the following browsers:

- [ ] JAWS Firefox Desktop
- [ ] JAWS Chrome Desktop
- [ ] NVDA Firefox Desktop
- [ ] NVDA Chrome Desktop
- [ ] VoiceOver Safari Desktop
- [ ] VoiceOver Chrome Desktop
- [ ] VoiceOver Safari Mobile
- [ ] Android Accessibility Suite Chrome Mobile

## Pull request type

Please check the type of change your PR introduces:

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by kyubisation — APPROVED** [review]
LGTM 👍 
Could this not also be configured via CSS?

**Comment by DavideMininni-Fincons** [comment]
> LGTM 👍 Could this not also be configured via CSS?

@kyubisation 
it could be set [here](https://github.com/lyne-design-system/lyne-components/blob/ff818b46d5565896574a0e1febee168bb28751bd/src/components/sbb-logo/sbb-logo.scss#L45) but i was not sure to apply them on every SVG (possible drawbacks?); if it's ok, I'll add the rules there too.

**Comment by kyubisation** [comment]
Since this is fairly static (we don't want to support other icons in the `sbb-logo` component), I think it would be ok to do it via CSS.

File: src/components/sbb-logo/sbb-logo.scss [hunk 1]
```diff
@@ -42,6 +42,8 @@
 }
 
 .sbb-logo svg {
+  width: 100%;
+  height: 100%;
   float: inline-start;
```
Comment (jeripeierSBB) [inline src/components/sbb-logo/sbb-logo.scss:47]: Maybe for a later rework: using float here seems not a good solution to me.

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Pull request merged**